app/serializers/balance_serializer.rb

```diff
@@ -9,6 +10,10 @@ def balance
     object.balance.to_s
   end
 
+  def pending_balance
+    object.pending_balance.to_s
```

Comment:
Why `to_s`? Serializer should by default stringify anything it processes.

Reply:
`pending_balance` is an instance of the `Money` class - `to_s` method simply returns human-readable balance.